snapshot/src/test/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksumTest.java

```diff
@@ -130,28 +130,36 @@ void shouldPersistChecksum() throws Exception {
     assertThat(actual.getCombinedValue()).isEqualTo(expectedChecksum.getCombinedValue());
   }
 
-  @DisabledIfEnvironmentVariable(named = "GITHUB_ACTIONS", matches = "true")
-  @EnabledOnOs(OS.LINUX)
+  /**
+   * This test is only enabled on CI as it relies on strace. You can run it manually directly if
+   * your system supports strace. See {@link io.camunda.zeebe.test.util.STracer} for more.
+   */
+  @EnabledIfEnvironmentVariable(named = "CI", matches = "true")
   @Test
   void shouldFlushOnPersist() throws Exception {
     // given
     final var traceFile = temporaryFolder.resolve("traceFile");
     final var expectedChecksum = SnapshotChecksum.calculate(multipleFileSnapshot);
     final var checksumPath = multipleFileSnapshot.resolveSibling("checksum");
-    final var tracer = STracer.traceFor(Syscall.FSYNC, traceFile);
+    final var tracer = STracer.tracerFor(Syscall.FSYNC, traceFile);
 
     // when
     try (tracer) {
       SnapshotChecksum.persist(checksumPath, expectedChecksum);
-    }
 
-    // then
-    STracerAssert.assertThat(tracer)
-        .fsyncTraces()
-        .hasSize(1)
-        .first(FSyncTraceAssert.factory())
-        .hasPath(checksumPath)
-        .isSuccessful();
+      // then
+      Awaitility.await("until fsync was called for our checksum file")
```

Comment:
:thought_balloon: This isn't ideal. I wish there was a way this could be rolled in with the API, but all my sketches were not very good (e.g. copy the RecordingExporter behavior), and it was just simpler and more flexible to use `Awaitility` - though possibly more error prone =/